packages/itwinui-css/src/workflow-diagram/workflow-diagram.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../stepper/index';
+
+@mixin iui-workflow-diagram {
+  @include iui-stepper;
```

Comment:
The `margin-bottom` gives the space between the stepper and the step label in the long stepper.  However I think this can be better achieved by apply `flex` and `gap`.  Will make this change.
![Screen Shot 2022-07-01 at 9 20 22 AM](https://user-images.githubusercontent.com/849817/176902761-02df6d27-1c55-41c4-baa2-75003310810d.png)